src/components/job-common.js

```diff
@@ -115,7 +115,7 @@ export const makeCromwellStatusLine = (executionStatus, backendStatus) => {
 export const makeSection = (label, children) => div({
   style: {
     flex: '0 0 33%', padding: '0 0.5rem 0.5rem', marginTop: '1rem',
-    whiteSpace: 'pre', textOverflow: 'ellipsis', overflow: 'hidden'
+    whiteSpace: 'pre', textOverflow: 'ellipsis'
```

Comment:
This was causing the status dropdown to become partially cut off when there were only a few rows. I don't believe that it has any adverse consequences

Reply:
Fixed in 8c6988f726aadfd166fd8b901d8098f741dca636

It didn't seem possible to have the child override the `overflow` property of the parent, so I allowed overriding the style in `makeSection`